camera1_topic: "cam1"
camera2_topic: "cam2"
camera3_topic: "cam4"
camera4_topic: "cam5"
camera5_topic: "cam5"

lidar1_topic: "cam1"
lidar2_topic: "cam1"
lidar3_topic: "cam1"
lidar4_topic: "cam1"

apx_topic: "apx"

# 表格刷新频率
print_hz: 1
# 期望的数据频率
camera_desired_hz: 30
lidar_desired_hz: 10
apx_desired_hz: 10
# 允许的频率抖动容差
tolerance: 0.8
# 平均滑窗的大小
sample_num: 20
# 同步丢失的timeout
sync_timeout: 0.2
# 未受到消息，切换NC的timeout
msg_timeout: 1.0